lib/Rendering.cpp

```diff
@@ -7,6 +7,121 @@
 #include "Evaluator.h"
 #include "Rendering.h"
 
+/*!
+ * \internal
+ *
+ * \brief The TileScreenPlacement class is a class that describes a tiles
+ * position and size within the viewport.
+ */
+struct TileScreenPlacement {
+    double pixelPosX;
+    double pixelPosY;
+    double pixelWidth;
+};
+
+/*!
+ * \internal
+ *
+ * \brief The TilePosCalculator class
+ * is a helper class for positioning tiles within the viewport.
+ */
+struct TilePosCalculator {
```

Comment:
Do we change TilePosCalculator to a `class`  instead of a `struct`? The documentation states it's a class, and since it has private fields, I think it's better to make it a class instead of a struct. I can sort this out, just let me know. @Didgy74